enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -168,14 +171,15 @@ $ curl -k -i -s -X POST -u admin:$PASSWORD https://$SUPERHUB/api/fr/setup-hub/su
 $ curl -k -i -s -X POST -u admin:$PASSWORD https://$FEEDER/api/fr/setup-hub/feeder
 ```
 
-### Enable feeder, without API
+### Enable feeder manually
 
 For older hubs:
 
 ```console
 $ ssh $CLOUDUSER$FEEDER
```

Comment:
Is there supposed to be an @ between user and feeder?

Reply:
`CLOUDUSER=root@`